src/kOS/Module/kOSVesselModule.cs

```diff
@@ -19,6 +19,7 @@ public class kOSVesselModule : VesselModule
         private bool initialized = false;
         private Vessel parentVessel;
         private bool hasRemoteTech = false;
+        private int counterRemoteTechRefresh = 0;
```

Comment:
Interesting, I haven't observed any issues like that on my end.  Something on your system must trigger the race more often than on mine.

We may need to tweak the value a little bit to ensure that we don't randomly find out that one tick is good enough for you, but two ticks might be common as well.  If 24 works for you, go for it.  If 23 or 22 doesn't give you the same aero effect issues, it might be a little more conservative of a choice.